src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/DSA/DSASignVerify.cs

```diff
@@ -29,9 +29,7 @@ public static void SignAndVerifyDataNew1024()
         {
             using (DSA dsa = DSAFactory.Create(1024))
             {
-                //SignAndVerify(DSATestData.HelloBytes, "SHA1", dsa.ExportParameters(true));
                 byte[] signature = dsa.SignData(DSATestData.HelloBytes, new HashAlgorithmName("SHA1"));
-                //Assert.Equal(48, signature.Length);
```

Comment:
Yeah, since the IEEE-whatever format gives us a defined length asserting it somewhere seems beneficial.